blog/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
 from .views import HomeView, PostDetailView, AddPostView, UpdatePostView,\
-    DeletePostView, AddCategoryView, CategoryView, UserRegisterView
+    DeletePostView, AddCategoryView, CategoryView, UserRegisterView, \
```

Comment:
For more readable code when you have many classed for import use tuple

Reply:
done